io.asgardeo.tomcat.oidc.agent/src/main/java/io/asgardeo/tomcat/oidc/agent/OIDCAgentFilter.java

```diff
@@ -171,6 +171,6 @@ private String buildErrorPageURL(OIDCAgentConfig oidcAgentConfig, HttpServletReq
         if (StringUtils.isNotBlank(oidcAgentConfig.getIndexPage())) {
             return oidcAgentConfig.getIndexPage();
         }
-        return request.getContextPath();
+        return SSOAgentConstants.DEFAULT_CONTEXT_ROOT;
```

Comment:
What's the value of SSOAgentConstants.DEFAULT_CONTEXT_ROOT?

Reply:
The value is set to "/"

Reply:
So of an error page not configured explicitly, we are redirecting the user to the application root path?

Reply:
Say the app context is my-app/.
If the `ErrorPage` property is not configured via the `app.properties` it would fallback to using the `indexPage` property.
The `indexPage` property has to be configured with the app context as in `my-app/index.html`. So, when fetching the request.getContextPath(), it would return /my-app and the error page would be set to my-app/my-app/index.html in case the errorPage property is not configured.

Reply:
> So if an error page not configured explicitly, we are redirecting the user to the application root path?

Yes